pkg/controllers/podSecurity/podsecurity.go

```diff
@@ -0,0 +1,49 @@
+package podsecurity
+
+import (
+	"context"
+	"time"
+
+	context2 "github.com/loft-sh/vcluster/cmd/vcluster/context"
+	"github.com/loft-sh/vcluster/pkg/util/loghelper"
+	corev1 "k8s.io/api/core/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	Namespace     = "kube-system"
+	ConfigMapName = "coredns"
+	NodeHostsKey  = "NodeHosts"
+)
+
+type PodSecurityReconciler struct {
+	Ctx *context2.ControllerContext
```

Comment:
use virtual client and podsecuritystandard here instead of the whole context, makes it a lot easier for unit testing